applications/nrf_desktop/doc/dfu.rst

```diff
@@ -132,6 +145,8 @@ The device generation allows to distinguish configurations that use the same boa
 reboot
    Perform the fetch operation on this option to trigger an instant reboot of the device.
 
+   With the :ref:`CONFIG_DESKTOP_DFU_LOCK <config_desktop_app_options>` option enabled, the device only executes this command when it holds the DFU owner status.
```

Comment:
We could use a shortcut for the repeated text. 
For example: https://github.com/nrfconnect/sdk-nrf/blob/main/applications/nrf_desktop/doc/battery_meas.rst#module-events